configure.ac

```diff
@@ -130,7 +130,7 @@ dnl arbitrary path - we don't actually install there.
      -DSYSCONF_INSTALL_DIR:PATH=/usr/libexec/rpm-ostree/etc \
      -DSHARE_INSTALL_PREFIX:PATH=/usr/libexec/rpm-ostree/share \
      -DBUILD_SHARED_LIBS:BOOL=ON \
-     ${cmake_args} ../libhif)
+     ${cmake_args} ../libhif) || exit 1
```

Comment:
I think the reason is that the resulting configure script itself doesn't have errexit on, so it doesn't care that the cmake subshell exited non-zero.